Packages/com.unity.inputsystem/InputSystem/Plugins/EnhancedTouch/Touch.cs

```diff
@@ -0,0 +1,498 @@
+        {
+            get
+            {
+                // We lazily construct the array of active fingers.
+                s_ActiveState.UpdateActiveFingers();
+                return new ReadOnlyArray<Finger>(s_ActiveState.activeFingers, 0, s_ActiveState.activeFingerCount);
+            }
+        }
+
+        public static IEnumerable<Touchscreen> screens => s_Touchscreens;
+
+        public static event Action<Finger> onFingerDown
+        {
+            add
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                s_OnFingerDown.AppendWithCapacity(value);
+            }
+            remove
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                var index = s_OnFingerDown.IndexOf(value);
+                if (index != -1)
+                    s_OnFingerDown.RemoveAtWithCapacity(index);
+            }
+        }
+
+        public static event Action<Finger> onFingerUp
+        {
+            add
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                s_OnFingerUp.AppendWithCapacity(value);
+            }
+            remove
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                var index = s_OnFingerUp.IndexOf(value);
+                if (index != -1)
+                    s_OnFingerUp.RemoveAtWithCapacity(index);
+            }
+        }
+
+        public static event Action<Finger> onFingerMove
+        {
+            add
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                s_OnFingerMove.AppendWithCapacity(value);
+            }
+            remove
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+                var index = s_OnFingerMove.IndexOf(value);
+                if (index != -1)
+                    s_OnFingerMove.RemoveAtWithCapacity(index);
+            }
+        }
+
+        /*
+        public static Action<Finger> onFingerTap
+        {
+            get { throw new NotImplementedException(); }
+            set { throw new NotImplementedException(); }
+        }
+        */
+
+        /// <summary>
+        /// The amount of history kept for each single touch.
+        /// </summary>
+        /// <remarks>
+        /// By default, this is zero meaning that no history information is kept for
+        /// touches. Setting this to <c>Int32.maxValue</c> will cause all history from
+        /// the beginning to the end of a touch being kept.
+        /// </remarks>
+        public static int maxHistoryLengthPerFinger
+        {
+            get => s_HistoryLengthPerFinger;
+            set { throw new NotImplementedException(); }
```

Comment:
comment out the setter if you can't use it.

Reply:
Done.